index.html

```diff
@@ -0,0 +1,180 @@
+            <img src="./assests/udle-2022-main.webp" alt="logo" srcset="" />
+          </a>
+          <div class="dropdown">
+            <div id="dropdownContent" class="dropdown-content">
+              <span class="close" onclick="closeDropdown()">&times;</span>
+              <a href="#home" onclick="closeDropdown()">Home</a>
+              <a href="./about.html" onclick="closeDropdown()">About</a>
+              <a href="#cards" onclick="closeDropdown()">Program</a>
+              <a href="#cards" onclick="closeDropdown()">Join</a>
+              <a href="#speakers" onclick="closeDropdown()">Speaker</a>
+            </div>
+          </div>
+          <div class="desk-ul">
+            <a href="#home" class="desk-li">News</a>
+            <a href="./about.html" class="desk-li">About</a>
+            <a href="#cards" class="desk-li">Program</a>
+            <a href="#cards" class="desk-li">Join</a>
+            <a href="#speakers" class="desk-li">Speaker</a>
+            <a href="#home" class="desk-li">ULD Compign</a>
+          </div>
+          <!-- mobile navigation endss here -->
+        </nav>
+      </header>
+      <!-- header starts here -->
+
+      <!-- home page starts here -->
+      <div id="home" class="home">
+        <div class="headline">
+          <h3>"Hello! Sharing World"</h3>
+          <h1>UNITING THE URBAN DESIGN AND LANDSCAPE EXPO 2023</h1>
+          <p class="headline-paragraph">
+            Urban Design & Landscape Expo is a prominent platform for the urban
+            design and landscape industry to expand your business in the Middle
+            East, Africa and South Asia. The event offers live product
+            showcases, knowledge exchange, and networking opportunities, all
+            alongside the prestigious Big 5 Global event.
+          </p>
+          <h2>2023.10.16(MON) ~ 19(THU)</h2>
+          <p class="foot-p">
+            @ DMG events is an organiser of face-to-face events and a publisher
+            of trade magazines for the international Construction, Energy,
+            Coatings, Transport, Hospitality & Design industries.
+          </p>
+        </div>
+      </div>
+    </section>
+    <!-- home page ends here -->
+    <!-- cards starts here -->
+    <section id="cards" class="cards">
+      <p>Main Program</p>
+      <hr class="about-hr" />
+      <div class="card-group">
+        <div class="card">
+          <img src="./assests/lecture.png" class="card-img-top" alt="card" />
+          <div class="card-body">
+            <h5 class="card-title">Lecture</h5>
+            <p class="card-text">
+              Listen to the speakers from the various countries about the
+              messages of sharing and opening
+            </p>
+          </div>
+        </div>
+        <div class="card">
+          <img src="./assests/Exhibition.png" class="card-img-top" alt="card" />
+          <div class="card-body">
+            <h5 class="card-title">CC Exhibition</h5>
```

Comment:
- [ ] Please perfect the main program section by adding the white border to the cards on hover, like in the original design.
- [ ] also, you are missing the `see all program` underlined text. Kindly add that.
yours
![Screenshot (1203)_LI](https://github.com/Kaiserabbas/Capstone/assets/65298048/e08f19e9-d28f-4335-b24c-185d1ca97e13)
Behance
![Screenshot (1204)](https://github.com/Kaiserabbas/Capstone/assets/65298048/9fb560bc-8a27-4ea9-81b9-be05774fea1c)